src/rules/at-import-no-partial-leading-underscore/index.js

```diff
@@ -16,6 +16,16 @@ const meta = {
 
 function rule(actual) {
   return (root, result) => {
+    result.warn(
```

Comment:
Looking at this, it seems that the call to `result.warn` should be placed to be after the `validOptions` check:
https://github.com/stylelint/stylelint/blob/13.7.0/lib/rules/at-rule-blacklist/index.js#L32C1-L35C6

Reply:
Right, thanks!